person.rb

```diff
@@ -0,0 +1,33 @@
+class Person
+  attr_accessor :name, :age, :parent_permission
+  attr_reader :id
+
+  def initialize(age, parent_permission, name = 'Unknown')
```

Comment:
- [ OPTIONAL ] Add the parent_permission as default true in your initialize method  `parent_permission = true`